src/flake8_simplify/plugins/yoda_conditions.rs

```diff
@@ -0,0 +1,34 @@
+use rustpython_ast::{Cmpop, Expr, ExprKind};
+
+use crate::ast::types::Range;
+use crate::checkers::ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+pub fn yoda_conditions(
```

Comment:
Mind just adding `/// SIM300` above this function? Helps a lot with grepping for rule implementations, since otherwise there's nothing tying this file to the check code.